NEWS.rst

```diff
@@ -3,6 +3,94 @@ http://twistedmatrix.com/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 22.1.0.rc1 (2022-01-26)
+===============================
+
+Features
+--------
+
+- support Python 3.10b4 (#10224)
+- Type annotations have been added to the twisted.python.fakepwd module. (#10287)
+
+
+Bugfixes
+--------
+
+- twisted.internet.defer.inlineCallbacks has an improved type annotation, to avoid typing errors when it is used on a function which returns a non-None result. (#10231)
+- revert changes to ``DelayedCall.__repr__`` and ``LoopingCall.__repr__`` (#10235)
+- restore .whl building (#10236)
+- SSHTransportBase.ssh_KEXINIT now uses the remote peer preferred MAC list for negotiation. In previous versions  it was only using the local preferred MAC list. (#10241)
+- The type annotation of the host parameter to twisted.internet.interfaces.IReactorTCP.connectTCP has been corrected from bytes to str. (#10251)
+- Deprecated ``twisted.python.threading.ThreadPool.currentThread()`` in favor of ``threading.current_thread()``.
+  Switched ``twisted.python.threading.ThreadPool.currentThread()`` and ``twisted.python.threadable.getThreadID()`` to use `threading.current_thread()`` to avoid the deprecation warnings introduced for ``threading.currentThread()`` in Python 3.10. (#10273)
+- twisted.web.client.RedirectAgent and twisted.web.client.BrowserLikeRedirectAgent now properly remove sensitive headers when redirecting to a different origin. (#10294)
```

Comment:
isn't this a Web bugfix?

Reply:
It is.  I will moved it. thanks

This is how it was classified during the PR and review process... so I just went with the default classification.

I hope in the future we can have a single "twisted" category so that we don't have to worry about web/mail/words/trial

Reply:
> I hope in the future we can have a single "twisted" category so that we don't have to worry about web/mail/words/trial

I hope not.  The user experience of reading 100 changes, where 20 web-related changes are randomly mixed with 80 non-web-related changes seems pretty bad.